src/parse-date.js

```diff
@@ -14,12 +62,12 @@ function parseDate(value) {
 
   if (isArray(value)) {
     return {
-      month: value[0],
+      month: value[0] || '',
```

Comment:
:+1: on defaulting this to an empty string to make the API consistent.

Reply:
no, if you pass an empty array then this value is undefined

```js
  if (/\//.test(value)) {
    value = value.split(/\s*\/\s*/g);
  } else if (/\s/.test(value)) {
    value = value.split(/ +/g);
  }

  if (isArray(value)) {
    return {
      month: value[0] || '',
      year: value.slice(1).join()
    };
  }
```

should I rule something here ?